### PR TITLE
Updated the scroll Bar of the event calendar and the side navbar.

### LINKED ISSUE #1551: Admin Redesign: custom scrollbar
**Is your feature request related to a problem? Please describe.**
we require a thin scrollbar in talawa to match the design consistency 

**Additional context**
To be implemented here across sidenavbar and calendar:
Screenshot 2024-02-06 at 10 25 33 PM
Screenshot 2024-02-06 at 10 26 00 PM

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by AmitSharma512**
**What kind of change does this PR introduce?**

I have updated the scroll bar of the event calendar and the side navbar.

**Issue Number:**

Fixes #1551 

**Snapshots/Videos:**

image

image

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@Kevoniat @anwersayeed Can you review this PR?

**Comment by AmitSharma512** [comment]
@aashimawadhwa @rishav-jha-mech, can you please review the PR ??

**Comment by palisadoes** [comment]
_**NOTE**_ Read very carefully

1. We just merged this PR which upgraded the prettier package. 
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/1599
1. It reformatted over 150 files. 

This will put your PR at risk of extensive merge conflicts. Do the following **IN THIS ORDER**:

1. upgrade your `prettier` in your local branch to the same version
1. run `prettier` on your local branch
1. update your local branch with the latest upstream
    
This will help to reduce the number of **existing** and **future** merge conflicts for your PR.

File: src/components/LeftDrawerOrg/LeftDrawerOrg.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { ReactComponent as TalawaLogo } from 'assets/svgs/talawa.svg';
 import styles from './LeftDrawerOrg.module.css';
 import { REVOKE_REFRESH_TOKEN } from 'GraphQl/Mutations/mutations';
 import useLocalStorage from 'utils/useLocalstorage';
+import '../../assets/css/scroll-styles.css';
```
Comment (rishav-jha-mech) [inline src/components/LeftDrawerOrg/LeftDrawerOrg.tsx:18]: @AmitSharma512 you need to import this into `/src/index.ts` file

File: src/assets/css/scroll-styles.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.custom-scroll {
```
Comment (rishav-jha-mech) [inline src/assets/css/scroll-styles.css:1]: make this `camelCase`, after this your PR is ready to be merged

**Comment by AmitSharma512** [comment]
@rishav-jha-mech can you please review it?

**Review by rishav-jha-mech — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**